CHANGELOG.md

```diff
@@ -1,5 +1,299 @@
 # Change Log
 
+See [here](./docs-md/operate-and-deploy/installation/upgrading.md) for upgrade notes.
```

Comment:
I would put this into a more prominent section, something like: 

## Upgrading
Note that ksqlDB 0.7.0 has a number of breaking changes when compared with ksqlDB 0.6.0 (see the 'Breaking changes' section below for details). Please make sure to read and follow these [upgrade instructions](./docs-md/operate-and-deploy/installation/upgrading.md) if you are upgrading from previous ksqlDB versions.